packages/cardano-services/src/Http/HttpServer.ts

```diff
@@ -65,6 +65,32 @@ export class HttpServer extends RunnableModule {
       this.app.use(`/${service.slug}`, service.router);
       this.logger.debug(`Using /${service.slug}`);
     }
+
+    const servicesHealthCheckHandler = async (req: express.Request, res: express.Response) => {
+      this.logger.debug('/health', { ip: req.ip });
+      let body: ServicesHealthCheckResponse | Error['message'];
+      try {
+        const servicesHealth = await Promise.all(
+          this.#dependencies.services.map((service) =>
+            service.healthCheck().then(({ ok }) => ({
+              name: service.name,
+              ok
+            }))
```

Comment:
Better be defensive, some bug in 1 healthcheck could mess up the entire `Promise.all`

```suggestion
            })).catch((err) => {
              logger.error(err);
              return { name: service.name, ok: false };
            })
```

Reply:
41fa605dd9cf00bedd1371e10ba716464835ed66

Reply:
After the applied change here (nested catch block) it turned out we never go into `unhealthy` state of Docker's `healthcheck`  request we send each 30s `curl --fail http://0.0.0.0:3000/health || exit 1`  never would fail as the catch shallows the actual error and failed response. 
This [catch](https://github.com/input-output-hk/cardano-js-sdk/pull/390/files#diff-db63771378e2c7af74fab64645861ab1f0556c3320ce46fabf60eedf796ab08eR73-R76) block will handle the failures. @mkazlauskas

Reply:
Shouldn't /health respond with 200 status regardless if it's healthy or not? cc @rhyslbw

Reply:
Yes. It's a successful HTTP request

Reply:
Addin the catch block https://github.com/input-output-hk/cardano-js-sdk/pull/390/commits/b3258cf016f6f82bcaf4e8ac8563e19b2c4772c2 @mkazlauskas